frontend/e-buddy/src/app/(demo)/bots/setup/page.tsx

```
@@ -1,153 +1,251 @@
+                  ? "Data Source"
+                  : "Configuration"}
               </span>
+
+              {/* Connector lines between steps */}
               {idx < 2 && (
-                <div className="flex-1 h-1 bg-gray-300 w-full mx-2 rounded-full" />
+                <div
+                  className={`step-connector ${
+                    step > s ? "step-connector-active" : ""
+                  }`}
+                ></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)